### PR TITLE
Update validators.he.xlf

### PR DESCRIPTION
**Pull request by ben29**
#38744

### CODE REVIEW

File: src/Symfony/Component/Form/Resources/translations/validators.he.xlf [hunk 1]
```diff
@@ -14,6 +14,126 @@
...
+                <source>Please enter a valid time.</source>
+                <target>אנא הזן שעה חוקי.</target>
+            </trans-unit>
+            <trans-unit id="120">
+                <source>Please select a valid timezone.</source>
+                <target>אנא בחר אזור זמן חוקי.</target>
+            </trans-unit>
+            <trans-unit id="121">
+                <source>Please enter a valid URL.</source>
+                <target>אנא הכנס כתובת תקינה.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.he.xlf:107]: Does this say "URL"? There is a difference between "address" and "URL".

File: src/Symfony/Component/Form/Resources/translations/validators.he.xlf [hunk 2]
```diff
@@ -14,6 +14,126 @@
...
+                <source>Please enter an integer.</source>
+                <target>אנא הזן מספר שלם.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>אנא בחר שפה חוקי.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>נא בחר אזור חוקי.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.he.xlf:75]: Does this refer to a locale like "en", "fr", "he" etc?

**Review by Nyholm — COMMENTED** [review]
Thank you. 

This is much appreciated. I have two small questions. I used google translate so I might be super wrong.

**Comment by ben29** [comment]
@Nyholm there is words you can’t 
Translate it to Hebrew
Let me fix this 2

**Pull request closed without merging**